darwin/future/meta/queries/item.py

```diff
@@ -143,3 +145,41 @@ def archive(self) -> None:
         ids = [item.id for item in self]
         filters = {"item_ids": [str(item) for item in ids]}
         archive_list_of_items(self.client, team_slug, dataset_ids, filters)
+
+    def tag(self, tag_id: str) -> None:
+        if "team_slug" not in self.meta_params:
+            raise ValueError("Must specify team_slug to query items")
+        if (
+            "dataset_ids" not in self.meta_params
+            and "dataset_id" not in self.meta_params
+        ):
+            raise ValueError("Must specify dataset_ids to query items")
+        dataset_ids = (
+            self.meta_params["dataset_ids"]
+            if "dataset_ids" in self.meta_params
+            else self.meta_params["dataset_id"]
+        )
+        team_slug = self.meta_params["team_slug"]
+        self.collect_all()
+        ids = [item.id for item in self]
+        filters = {"item_ids": [str(item) for item in ids]}
+        tag_items(self.client, team_slug, dataset_ids, tag_id, filters)
+
+    def untag(self, tag_id: str) -> None:
+        if "team_slug" not in self.meta_params:
+            raise ValueError("Must specify team_slug to query items")
+        if (
+            "dataset_ids" not in self.meta_params
+            and "dataset_id" not in self.meta_params
+        ):
+            raise ValueError("Must specify dataset_ids to query items")
```

Comment:
As above
```suggestion
            raise ValueError("Must specify dataset_ids to query items")
Similar to the other Meta methods(i.e. `move_to_folder`), I'd add a check for the `tag_id` arg here like:
```suggestion
            raise ValueError("Must specify dataset_ids to query items")
        if not tag_id:
            raise ValueError("Must specify tag_id to tag items with")
```   
```